packages/grid/_modules_/grid/components/menu/GridMenu.tsx

```diff
@@ -52,11 +65,20 @@ export const GridMenu: React.FC<MenuProps> = ({
   }, [open, target]);
 
   return (
-    <Popper open={open} anchorEl={target as any} transition placement={position} {...other}>
+    <Popper
+      className={classes.root}
+      open={open}
+      anchorEl={target as any}
+      transition
+      placement={position}
+      {...other}
+    >
       {({ TransitionProps, placement }) => (
         <Grow {...TransitionProps} style={{ transformOrigin: transformOrigin[placement] }}>
           <Paper>
-            <ClickAwayListener onClickAway={onClickAway}>{children}</ClickAwayListener>
+            <ClickAwayListener onClickAway={onClickAway}>
+              <div>{children}</div>
```

Comment:
I think that it should aways be a MenuList as a first child (can be coming from children). Why is it not working in HEAD?

Reply:
It's weird, it works for the export and the density but it doesn't for the column menu, by not working I mean it throws a warning in the console.

Reply:
Ok, I had a look. Any weird behavior is an invitation for exploration. The `GridColumnMenu` component isn't forwarding its ref, not the extra props to the `MenuList` component. It could be solved easily but, there a bunch of extra props (the `GridBaseComponentProps`) and it's not versatile.

Instead, I have pushed a commit to reorganize the composition order. Grow doesn't host any div, it needs to apply props to its children, same for ClickAwayListener. We save one DOM node.

In the process, I have found an improvement opportunity, ClickAwayListener could forward its props to the children.